pkg/xds/generator/testdata/direct-access/01.dataplanes.input.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I guess these are ok to be empty?

Reply:
Yeah, it does not matter since it is not executed, because transparent proxy is off